tool/test/org/antlr/v4/test/tool/TestLexerErrors.java

```diff
@@ -30,7 +31,7 @@ public void testStringsEmbeddedInActions() {
 		String grammar =
 			"lexer grammar Actions;\n"
 			+ "ACTION2 : '[' (STRING | ~'\"')*? ']';\n"
-			+ "STRING : '\"' ('\\\"' | .)*? '\"';\n"
+			+ "STRING : '\"' ('\\\\\"' | .)*? '\"';\n"
```

Comment:
➡️ Previous test was behaving as though `STRING` was defined as follows, which was not the intent:

```antlr
STRING : '\"' ('"' | .)*? '\"';
```